src/main/java/mat/client/export/measure/ManageMeasureExportView.java

```diff
@@ -127,6 +128,17 @@ public void setExportOptionsBasedOnVersion(String releaseVersion, boolean isComp
 		vp.add(compositeMeasurePackageRadio);
 		resetRadioButtonValues(isCompositeMeasure);
 	}
+
+	private boolean isV5OrGreater(String version) {
+		boolean result = false;
+		try {
+			result = Double.parseDouble(version.substring(1,version.length())) >= 5.0;
+		} catch (RuntimeException re) {
```

Comment:
`RuntimeException` seems overly broad. Could it be some combination of other exceptions? If we get up to a runtime exception, there's probably something else we should be doing.

Reply:
It handles all the cases I need NullPointer,NumberFormat, etc. Why create multiple blocks and increase the code size?

Reply:
It's java 11.  catch(NullPointer|IllegalArgumentException e) { // ... }
Indeed. Most of code scanning toold will complain about a borad exception :)

Reply:
nice, switched over to java11 style.

Reply:
sorry for misleading you.
it's a Java7 feature
https://docs.oracle.com/javase/7/docs/technotes/guides/language/catch-multiple.html